benchmarks/tests/primary/src/helpers.js

```diff
@@ -0,0 +1,132 @@
+import { dirname } from "node:path";
+import { fileURLToPath } from "node:url";
+
+
+export function neverOptimizeFunction(f) {
+    %NeverOptimizeFunction(f);
+    return f;
+}
+
+// dprint-ignore-file
+export function gc(iters=1) {
```

Comment:
Can you explain why more iters would be useful here?

Reply:
Around the web there are varying positions about this. But basically triggering a gc isn't instant from what I can tell. There are also a ton of things happening in the background too. At startup I noticed a lot of variability in how much was collected and this affected the heap diffs. 

Originally I was just running this 10 times in various forms but needed more on startup it seemed. But didn't need the delay at the end as the numbers didn't change much from 1 iteration to 100 later on. 

I assume this is related to internal optimizations of the parse/compile/hit phases as at some point huge amounts of memory get freed up.